src/lib/viewers/BaseViewer.js

```diff
@@ -1257,7 +1254,21 @@ class BaseViewer extends EventEmitter {
         // we remain in create mode
         if (status === 'success') {
             this.annotator.emit('annotations_active_set', id);
+
+            if (this.annotationControls) {
+                this.annotationControls.setMode(
+                    this.annotationControlsFSM.transition(AnnotationInput.SUCCESS, AnnotationMode.NONE),
```

Comment:
Is the second argument of `transition` required for any of the staged change event transitions?